airbyte-integrations/connectors/source-appstore-singer/source_appstore_singer/spec.json

```diff
@@ -25,7 +25,7 @@
       },
       "start_date": {
         "type": "string",
-        "description": "date from which to start pulling data",
+        "description": "date from which to start pulling data (example: 2020-11-16T00:00:00Z)",
```

Comment:
@sherifnada can we make sure that examples like this are shown in the description string instead of examples which aren't shown in the UI? I imagine it's hard for most people to go backwards from the pattern error to what the output should look like.

Reply:
Sure, not a problem. But how come we don't show `examples` in the UI? Wouldn't that be the more intuitive DX?

Reply:
Actually you're right. I'm going to remove this from the description and just make a high priority frontend ticket to show the examples. I imagine this is really confusing right now.

Reply:
https://github.com/airbytehq/airbyte/issues/1976